ktor-features/ktor-metrics/src/io/ktor/metrics/dropwizard/DropwizardMetrics.kt

```diff
@@ -8,7 +8,7 @@ import io.ktor.routing.*
 import io.ktor.util.*
 import java.util.concurrent.*
 
-class Metrics(val registry: MetricRegistry) {
+class DropwizardMetrics(val registry: MetricRegistry) {
```

Comment:
A deprecated typealias with replacement configured is required here for smoother migration.

Reply:
So this module will replace the Dropwizard one? I was told by @orangy to make 2 modules...

Reply:
No, just to keep already existing code working. Something like this:

```kotlin
package package io.ktor.metrics

typealias Metrics = io.ktor.metrics.dropwizard.DropwizardMetrics
```

Reply:
done

Reply:
It will not compile. Deprecated annotation requires a message. replaceWith should be an expression string.

Reply:
that was a typical "it (kind of) works on my machine" there was an error from the rebase so the file was in the wrong directory and therefore not compiled and gradle did not return an error...
good finding